examples/common/maths.adb

```diff
@@ -206,6 +204,45 @@ package body Maths is
     end Scaling_Matrix;
 
     --  ------------------------------------------------------------------------
+
+    function To_3 (Array_4 : Single_Array) return Single_Array is
+        Array_3  : Single_Array (0 .. Array_4'Length * 3 / 4 - 1);
```

Comment:
My understanding is that OpenGL arrays, particularly vertex arrays start at 0.
I agree that Ada  arrays start at 1 by convention and this is certainly my preference but I assume that GLSL shaders expect arrays to start at 0.
Any way, as I said above I'll remove these functions from Maths until and if I can fix them .
By the way, I have just updated my OS to Sierra.